README.md

```diff
@@ -487,7 +498,51 @@ app.addRoute('/404', noPage)
 
 </details>
 
-### `app.start(selector, [pathName])`
+<details>
+<summary>
+Nested Routes Example:
+</summary>
+
+```js
+/* index.js */
+const app = new Tram()
+const html = Tram.html()
+const route = Tram.route()
+
+const homePage = (store, actions, params, child) => {
```

Comment:
Are all of these parameters being used?

Reply:
no, but the last one is, which means they all have to be written out.

Reply:
You could also use _ correct?

Reply:
Yes, but I would argue as an example it's good to repeat the API.